gobiko/apns/client.py

```diff
@@ -35,7 +35,7 @@
     'InternalServerError': 500, 
     'ServerUnavailable': 503,
 }
-APNSResponseStruct = namedtuple('APNSResponseStruct', ' '.join(APNS_RESPONSE_CODES.keys()))
+APNSResponseStruct = namedtuple('APNSResponseStruct', APNS_RESPONSE_CODES.keys())
```

Comment:
`namedtuple` accepts a sequence of strings as list of names, doing `' '.join()` is unnecessary